src/main/java/asu/eng/gofund/model/Comment.java

```diff
@@ -0,0 +1,142 @@
+package asu.eng.gofund.model;
+
+import jakarta.persistence.*;
+
+import java.sql.Date;
+import java.util.List;
+
+@Entity
+@Table(name = "comment")
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private boolean isDeleted;
+    private String content;
+
+    @OneToOne
+    private Person author;
+    private Date timestamp;
+    @ManyToOne
+    @JoinColumn(name = "id")
+    private Campaign campaign;
+    @ManyToOne
+    @JoinColumn(name = "parent_id")
+    private Comment parentComment;
+    @OneToMany(mappedBy = "parentComment", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Comment> replies;
+    private boolean edited;
+
+    public Comment(){}
+
+    public Comment(long id,
```

Comment:
Add a third constructor for Comment, with all attributes except ID
Thsi should be used when we try to insert as the object doesn't have an id yet